pkg/k8s/init.go

```diff
@@ -139,7 +141,7 @@ func Init(conf k8sconfig.Configuration) error {
 	if err != nil {
 		return fmt.Errorf("unable to create k8s client rest configuration: %s", err)
 	}
-	closeAllDefaultClientConns := setDialer(restConfig)
```

Comment:
This line should not be removed, the `restConfig` is being modified and the result of it is being used on the following lines. If we don't modify the `restConfig` here then the none of the clients will use the dialer set by `setDialer`

Reply:
Fixed, good catch.

Reply:
It's not fixed, `getCloseAllConnsFunc(restConfig)` is only executed on line 186. `setDialer` needs to be executed on this line.

Reply:
Should be good now